### PR TITLE
fix(spec): do not include unpackaged rendezvous-info.yml file

### PR DESCRIPTION
**Pull request by 7flying**
Removes rendezvous-info.yml from the install list for the aio command,
the contents of this file are already embedded into the manufacturing-server.yml file,
so aio does not need it.

This fixes the following error when trying to `make rpm` locally:
```
Checking for unpackaged file(s): /usr/lib/rpm/check-files fido-device-onboard-rs/rpmbuild/BUILDROOT/fido-device-onboard-0.4.7-1.20230207gitdaa4a6c.el9.x86_64
error: Installed (but unpackaged) file(s) found:
   /usr/share/doc/fdo/rendezvous-info.yml
```

### CODE REVIEW

**Comment by runcom** [comment]
should we then just remove the file altogether if it's merged into the manufacturing config and hence unused?

**Comment by 7flying** [comment]
> should we then just remove the file altogether if it's merged into the manufacturing config and hence unused?

No because it is referenced in the HOWTO. You need this file in order to generate an OV and a device_credential, so we should provide with an example.

**Review by nullr0ute — COMMENTED** [review]
Can we drop the fdo and space so it's just %{buildroot}%{_docdir}/examples/ (also no spaces).

**Comment by 7flying** [comment]
> Can we drop the fdo and space so it's just %{buildroot}%{_docdir}/examples/ (also no spaces).

Let me know if the latest change makes sense

File: fido-device-onboard.spec [hunk 1]
```diff
@@ -70,7 +70,11 @@ install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-admin-to
 install -D -m 0755 -t %{buildroot}%{_bindir} target/release/fdo-owner-tool
 install -D -m 0755 -t %{buildroot}%{_bindir} target/release/fdo-admin-tool
 install -D -m 0644 -t %{buildroot}%{_unitdir} examples/systemd/*
-install -D -m 0644 -t %{buildroot}%{_docdir}/fdo examples/config/*
+# we do not need the rendezvous-info.yml for the AIO command, add everything else
+install -D -m 0644 -t %{buildroot}%{_docdir}/examples examples/config/manufacturing-server.yml
```
Comment (nullr0ute) [inline fido-device-onboard.spec:74]: you now have two examples and still a space :) so basically "sed -i 's/examples //' fido-device-onboard.spec"

File: fido-device-onboard.spec [hunk 2]
```diff
@@ -103,8 +107,8 @@ Requires: openssl-libs >= 3.0.1-12
 %{_mandir}/man1/fdo-owner-onboarding-server.1*
 %{_libexecdir}/fdo/fdo-owner-onboarding-server
 %{_libexecdir}/fdo/fdo-serviceinfo-api-server
-%{_docdir}/fdo/owner-onboarding-server.yml
-%{_docdir}/fdo/serviceinfo-api-server.yml
+%{_docdir}/examples/owner-onboarding-server.yml
+%{_docdir}/examples/serviceinfo-api-server.yml
```
Comment (nullr0ute) [inline fido-device-onboard.spec:111]: but, yes I think we need this to actually be named %{_docdir}/%{name}/examples/ and same for the ones below.

File: fido-device-onboard.spec (hunk 1 above)
Comment (7flying) [reply fido-device-onboard.spec:74]: The only space that I can see is the space that separates the <dest-dir> from the <source-dir>, what other space is there that I can't see?

Comment (nullr0ute) [reply fido-device-onboard.spec:74]: Yep, you're right, I was reading it wrong :-/

**Review by nullr0ute — APPROVED** [review]

File: fido-device-onboard.spec (hunk 1 above)
Comment (7flying) [reply fido-device-onboard.spec:74]: No problem!

**Pull request merged**